apps/molgenis-components/src/components/utils.ts

```diff
@@ -61,6 +62,52 @@ export function getPrimaryKey(
   }
 }
 
+export async function convertRowToPrimaryKey(
+  row: IRow,
+  tableName: string,
+  schemaName: string
+): Promise<Record<string, any> | null> {
+  const client = Client.newClient(schemaName);
+  const tableMetadata = await client.fetchTableMetaData(tableName);
+  if (!tableMetadata?.columns) {
+    return null;
+  } else {
```

Comment:
Throw error instead of null in order to make the return type a bit easier to deal with?